website/public/js/controllers/menuCtrl.js

```diff
@@ -41,7 +45,7 @@ angular.module('habitrpg')
       };
 
       $scope.hasNoNotifications = function() {
-        return selectNotificationValue(false, false, false, false, true);
+        return selectNotificationValue(false, false, false, false, false, true);
```

Comment:
Yeah, bit of a "lolwut" when I saw that. Not built in extensible fashion, then extended anyway.